static/carousel/css/styles.css

```diff
@@ -47,7 +47,6 @@ body  {
   right: 0;
   position: absolute;
   margin: auto;
-  -webkit-transition: all .2s ease;
```

Comment:
this was from running grunt, i don't know why it removed those, i'm taking a look